src/wrapper_array.jl

```diff
@@ -35,7 +35,7 @@ struct NamedDimsArray{L, T, N, A<:AbstractArray{T, N}} <: AbstractArray{T, N}
     data::A
 end
 
-function NamedDimsArray{L}(orig::AbstractArray{T, N}) where {L, T, N}
+@inline function NamedDimsArray{L}(orig::AbstractArray{T, N}) where {L, T, N}
```

Comment:
Did this sneak in or is it related to allocations?

Reply:
It is related to allocations.
Without this, it allocates